starlette/responses.py

```diff
@@ -105,7 +106,7 @@ def set_cookie(
         key: str,
         value: str = "",
         max_age: typing.Optional[int] = None,
-        expires: typing.Optional[int] = None,
+        expires: typing.Optional[typing.Union[datetime, str]] = None,
```

Comment:
Let's add `int` back. It's working with it, even if it's not in any standard that I've found.

The question now becomes if we should accept `str`. Because objectively speaking, we can just accept a `datetime`, and it's already enough to solve the issue.

Reply:
Sure thing. I will update the typing